autoload/ctrlsf/win.vim

```diff
@@ -119,6 +119,13 @@ func! s:InitMainWindow() abort
     setl textwidth=0
     setl nospell
     setl nofoldenable
+    if exists("*g:CtrlSFCustomizeWindowFunction")
```

Comment:
Maybe renaming this variable to `g:CtrlSFAfterMainWindowInit` should be better? We may add callback of `g:CtrlSFAfterPreviewWindowInit` or `g:CtrlSFBeforeMainWindowInit` or something else.